src/Symfony/Component/Notifier/Bridge/Clickatell/ClickatellTransport.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Clickatell;
+
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Kevin Auvinet <k.auvinet@gmail.com>
+ *
+ * @experimental in 5.3
+ */
+final class ClickatellTransport extends AbstractTransport
+{
+    protected const HOST = 'api.clickatell.com';
+
+    private $authToken;
+    private $from;
+
+    public function __construct(string $authToken, ?string $from = null, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->authToken = $authToken;
+        $this->from = $from;
+
+        parent::__construct($client, $dispatcher);
+    }
+
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof SmsMessage) {
+            throw new UnsupportedMessageTypeException($this, SmsMessage::class, $message);
```

Comment:
```suggestion
            throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
```

This needs to be a string